lib/java_buildpack/container/tomee/tomee_instance.rb

```diff
@@ -34,6 +44,21 @@ def tomcat_7_compatible
         @version < TOMEE_7
       end
 
+      private
+
+      def web_inf_lib
+        if ear?
+          # copy additional libraries to tomee lib folder
+          tomcat_lib
```

Comment:
Oops I think I misunderstood your recommendation @violetagg, what I did was to link stuff under `drivers` folder of ear to tomee/lib, however it works out well without needing `common.loader` change, how about just continuing this way since each instance of tomee is not going to host more than 1 ear anyway?